### ISSUE TITLE
Add support for modified timestamps in notes sorting & metadata displays

### ISSUE THREAD

**Issue by marcus-grant** [marcus-grant/brain2#4]
It's too time consuming & complex to write down the timestamps for modification dates by hand. And one of the best ways to review notes is to find the ones most recently modified.
This isn't trivial. Potentially it's possible to use the `gatsby-source-filesystem` plugin to get modified timestamps straight from the filesystem. However, as these [two](https://angelos.dev/2019/09/add-support-for-modification-times-in-gatsby/) [articles](https://joshuatz.com/posts/2019/gatsby-better-last-updated-or-modified-dates-for-posts/) it might be preferrable to use git's *author time* metadata instead as it has the dates of file modification diffs for each commit.

TLDR; Explore these options in the two above articles and try to create a node API update and associated queries and components to firstly, sort notes in the index by modification date and secondly display the modification date in the notes and index.

**Comment by marcus-grant** [comment]
This seems like the best solution so far. Currently there's a branch ft/git-mtime to work on this but I think fixing the slugs queries to be more important right now

**Comment by marcus-grant** [comment]
This https://github.com/joshuatz/git-date-extractor repo has some potential solutions

**Comment by marcus-grant** [comment]
Will continue sometime after #13 is sorted.
